.github/workflows/deployment.yaml

```diff
@@ -30,3 +30,11 @@ jobs:
             npm i
           fi
           yarn deploy
+      - name: Generate Seneca-IPC144.pdf
```

Comment:
Move this before  you call `deploy` so that it can already be included in the `static/` dir.